acceptance/tests/load_libfacter.rb

```diff
@@ -1,31 +1,44 @@
-test_name 'Ruby can load libfacter'
+test_name 'C100161: Ruby can load libfacter' do
```

Comment:
After reviewing this again, I do not understand the core intention of this test. It seems concerned with the low level plumbing of how the puppet-agent ruby file are laid out rather than validating functionality. Is there a functional side effect of not being able to load libfacter via ruby? If there is can we test that instead?